src/FixedDataTableBufferedRows.js

```diff
@@ -50,7 +50,6 @@ var FixedDataTableBufferedRows = React.createClass({
   },
 
   componentDidMount() {
-    setTimeout(this._updateBuffer, 1000);
```

Comment:
Hmm it looks like the method does exist in the file.   Rather than remove this, I think we need to figure out why it's undefined in this method call.  Maybe something bad with how the this object is bound?